partcad/tests/unit/test_project.py

```
@@ -9,6 +9,8 @@
 #
 
 import os
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)